pkgs/development/python-modules/myst-docutils/default.nix

```diff
@@ -33,6 +33,8 @@ buildPythonPackage rec {
     hash = "sha256-QbFENC/Msc4pkEOPdDztjyl+2TXtAbMTHPJNAsUB978=";
   };
 
+  patches = [ ./fix-amsmath-test.patch ];
```

Comment:
Have you tried using `fetchpatch` instead? https://github.com/NixOS/nixpkgs/tree/master/pkgs#patches

Reply:
No, will give it a try now! :+1:

Reply:
Updated it to use `fetchpatch`. Thanks for the suggestion!